--- liblo.c.orig	2007-08-03 12:04:59.000000000 +0200
+++ liblo.c	2007-08-03 12:07:27.000000000 +0200
@@ -114,9 +114,9 @@
 static PyObject *__pyx_k16;
 static lo_timetag (__pyx_f_5liblo__double_to_timetag(double )); /*proto*/
 static double (__pyx_f_5liblo__timetag_to_double(lo_timetag )); /*proto*/
-static int (__pyx_f_5liblo__callback(char (*),char (*),lo_arg (*(*)),int ,lo_message ,void (*))); /*proto*/
-static int (__pyx_f_5liblo__callback_threaded(char (*),char (*),lo_arg (*(*)),int ,lo_message ,void (*))); /*proto*/
-static void (__pyx_f_5liblo__err_handler(int ,char (*),char (*))); /*proto*/
+static int (__pyx_f_5liblo__callback(const char (*),const char (*),lo_arg (*(*)),int ,lo_message ,void (*))); /*proto*/
+static int (__pyx_f_5liblo__callback_threaded(const char (*),const char (*),lo_arg (*(*)),int ,lo_message ,void (*))); /*proto*/
+static void (__pyx_f_5liblo__err_handler(int ,const char (*),const char (*))); /*proto*/
 
 
 /* Implementation of liblo */
@@ -841,7 +841,7 @@
 static PyObject *__pyx_n_ismethod;
 
 
-static int __pyx_f_5liblo__callback(char (*__pyx_v_path),char (*__pyx_v_types),lo_arg (*(*__pyx_v_argv)),int __pyx_v_argc,lo_message __pyx_v_msg,void (*__pyx_v_cb_data)) {
+static int __pyx_f_5liblo__callback(const char (*__pyx_v_path),const char (*__pyx_v_types),lo_arg (*(*__pyx_v_argv)),int __pyx_v_argc,lo_message __pyx_v_msg,void (*__pyx_v_cb_data)) {
   unsigned char (*__pyx_v_ptr);
   uint32_t __pyx_v_size;
   uint32_t __pyx_v_j;
@@ -1262,7 +1262,7 @@
   return __pyx_r;
 }
 
-static int __pyx_f_5liblo__callback_threaded(char (*__pyx_v_path),char (*__pyx_v_types),lo_arg (*(*__pyx_v_argv)),int __pyx_v_argc,lo_message __pyx_v_msg,void (*__pyx_v_cb_data)) {
+static int __pyx_f_5liblo__callback_threaded(const char (*__pyx_v_path),const char (*__pyx_v_types),lo_arg (*(*__pyx_v_argv)),int __pyx_v_argc,lo_message __pyx_v_msg,void (*__pyx_v_cb_data)) {
   PyGILState_STATE __pyx_v_gil;
   int __pyx_r;
 
@@ -1286,7 +1286,7 @@
 
 static PyObject *__pyx_n___exception;
 
-static void __pyx_f_5liblo__err_handler(int __pyx_v_num,char (*__pyx_v_msg),char (*__pyx_v_where)) {
+static void __pyx_f_5liblo__err_handler(int __pyx_v_num,const char (*__pyx_v_msg),const char (*__pyx_v_where)) {
   PyObject *__pyx_1 = 0;
   PyObject *__pyx_2 = 0;
   PyObject *__pyx_3 = 0;
